azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -24,13 +24,13 @@
     private String environment;
     /**
      * Registered application ID in Azure AD.
+     * @NotEmpty only required when OAuth2 authentication is done in front end
```

Comment:
What are those `@NotEmpty` annotation in the Javadoc description?

Reply:
Moved the previous annotation directly to comments as the requirement changed..

Will update the Javadoc comment.